tests/positive/general/preemption.go

```diff
@@ -75,7 +75,15 @@ func makePreemptTest(components string) types.Test {
 
 	var systemFiles []types.File
 	for _, component := range []string{"b", "u"} {
-		if enabled(component) {
+		if enabled(component) && strings.Compare(component, "b") == 0 {
+			systemFiles = append(systemFiles, types.File{
+				Node: types.Node{
+					Name:      longnames[component] + ".ign",
+					Directory: "base.d",
+				},
+				Contents: makeConfig(component),
+			})
+		} else {
 			systemFiles = append(systemFiles, types.File{
```

Comment:
We're now doing this even if the component is disabled.

Reply:
Agreed! I've addressed this issue in the subsequent push.